setup/pkg/fieldlabs/environment_create.go

```diff
@@ -566,7 +566,7 @@ func (e *EnvironmentManager) createRBAC(envs []Environment, policies map[string]
 			return errors.Wrap(err, "marshal definition body")
 		}
 		rbacBody := &Policy{
-			Name:        e.getAppName(env),
+			Name:        env.App.ID,
 			Description: e.getAppName(env),
```

Comment:
I'd change this back to `e.getAppName(env)` so the name and description for which user is the same in the UI.